src/site/xdoc/checks/coding/illegaltype.xml

```diff
@@ -670,6 +670,40 @@ public class Example7 extends TreeSet {
   final Consumer&lt;Foo&gt; consumer = Foo&lt;Boolean&gt;::foo;
 
 }
+</code></pre></div>
+        <p id="Example8-config">
+          To configure the check so that it adds a type to the default illegal class names:
+        </p>
+        <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
+&lt;module name="Checker"&gt;
+  &lt;module name="TreeWalker"&gt;
+    &lt;module name="IllegalType" /&gt;
```

Comment:
remove this line, only single instance of Check is required.

Reply:
Done